src/Schema/Values/ContextModule.ts

```diff
@@ -151,6 +153,7 @@ export enum ContextModule {
   trendingArtistsRail = "trendingArtistsRail",
   trendingLots = "trendingLots",
   upcomingAuctions = "upcomingAuctions",
+  uploadPhotosCompleted = "uploadPhotosCompleted",
```

Comment:
**suggestion:** Is it OK if these context modules get reused for tracking other events, like uploading photos for My Collection, or entering contact information on your user profile? If not, I think we should group them together so that it's clear that they belong to the Sell with Artsy submission flow.

Reply:
Yes it's okay if they get reused elsewhere!